pyvo/registry/regtap.py

```diff
@@ -352,12 +352,16 @@ class _BrowserService:
     services.
     """
 
-    def __init__(self, access_url):
-        self.access_url = access_url
```

Comment:
This would be an API change

Reply:
Yes. I still think it's a useful one cause all the Services classes have a `baseurl` attribute except this one. May I keep the two with a deprecation warning on `access_url`?

Reply:
Yes, you're right I haven't noticed the private nature of the class. Please ignore my comment, renaming the attribute (and argument) is totally fine the way you originally have in the PR.
I've double-checked the usage everywhere else in the code and didn't find any issues with it either.